src/app/components/mnoe-api/app-instances.svc.coffee

```diff
@@ -92,4 +92,16 @@ angular.module 'mnoEnterpriseAngular'
       else
         "INSTALLED_LAUNCH"
 
+    @getAppInstanceSync = ->
+      defer = $q.defer()
```

Comment:
I don't think we need to use `$q.defer` here. And the nested promise is just resolving the response.

> If the returned value is a promise, then the further execution is suspended until it settles. After that, the result of that promise is given to the next .then handler.
-- _https://javascript.info/promise-chaining#returning-promises_

So we could just have:
```coffeescript
@getAppInstanceSync = ->
  MnoeOrganizations.get().then(
   -> _self.appInstancesSyncPromise = MnoeApiSvc.one('organizations', MnoeOrganizations.selectedId).one('/app_instances_sync').get()
  )
```
Disclaimer: double check that it works as advertised  ;)